lib/vm.js

```diff
@@ -97,6 +100,19 @@ function getContextOptions(options) {
   return {};
 }
 
+function isContext(sandbox) {
+  if (arguments.length < 1) {
+    throw new ERR_MISSING_ARGS('sandbox');
+  }
+
+  if (typeof sandbox !== 'object' && typeof sandbox !== 'function' ||
```

Comment:
I'm not sure we want to accept functions?

Reply:
I believe that’s just for compatibility: the original `->IsObject()` test accepts functions.